src/components/checkboxes/without-heading/index.njk

```diff
@@ -0,0 +1,34 @@
+---
+title: Checkboxes without a heading
+layout: layout-example.njk
+---
+
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+
+{{ govukCheckboxes({
+  idPrefix: "waste",
+  name: "waste",
+  fieldset: {
+    legend: {
+      text: "Which types of waste do you transport?",
+      isPageHeading: false
```

Comment:
We don't need to set `isPageHeading` here as it defaults to `false` (we've got an example [here](https://github.com/alphagov/govuk-frontend/blob/master/app/views/examples/labels-legends-and-headings/index.njk#L599)).